pygameMenu/baseimage.py

```diff
@@ -30,21 +30,33 @@
 -------------------------------------------------------------------------------
 """
 
-import os.path as _path
+import os.path as path
 import pygame
+import math
+
+from pygameMenu.utils import assert_vector2
 
 # https://www.pygame.org/docs/ref/image.html
 _VALID_IMAGE_FORMATS = ['.jpg', '.png', '.gif', '.bmp', '.pcx', '.tga', '.tif', '.lbm',
                         '.pbm', '.pgm', '.ppm', '.xpm']
 
 # Available images
-__actualpath = str(_path.abspath(_path.dirname(__file__))).replace('\\', '/')
+__actualpath = str(path.abspath(path.dirname(__file__))).replace('\\', '/')
 __fontdir = '{0}/resources/images/{1}'
 
-IMAGE_CARBON_FIBER = __fontdir.format(__actualpath, 'carbon_fiber.png')
-IMAGE_GRAY_LINES = __fontdir.format(__actualpath, 'gray_lines.png')
-IMAGE_METAL = __fontdir.format(__actualpath, 'metal.png')
-IMAGE_PYGAME_MENU = __fontdir.format(__actualpath, 'pygame_menu.png')
+# Example images
+IMAGE_EXAMPLE_CARBON_FIBER = __fontdir.format(__actualpath, 'carbon_fiber.png')
+IMAGE_EXAMPLE_GRAY_LINES = __fontdir.format(__actualpath, 'gray_lines.png')
+IMAGE_EXAMPLE_METAL = __fontdir.format(__actualpath, 'metal.png')
+IMAGE_EXAMPLE_PYGAME_MENU = __fontdir.format(__actualpath, 'pygame_menu.png')
+
+# Drawing modes
+IMAGE_MODE_CENTER = 0
```

Comment:
The integer values of  `IMAGE_MODE...`are not unique, same values used for `MENUBAR_STYLE...`

Reply:
Fixed!